### ISSUE TITLE
Data generation process

### ISSUE THREAD

**Issue by davasli** [FANGAreNotGnu/ControlAug#2]
Hi, 

I'm trying to understand how your approach is learning the data distribution for data generation? In a deep dive to the code, I don't see where the model learns the data distribution for synthesis purposes. If you could provide some insights on which script does the learning  that will be great. 

Thanks,
Kemal

**Comment by FANGAreNotGnu** [comment]
Hi Kemal,

In our paper, we utilize the diffusion model without additional training. I experimented with finetuning the diffusion model to better match the data distribution, but found several limitations: (1) it's computationally expensive and time-consuming, (2) it doesn't work well for few-shot or small dataset scenarios, (3) there's a high risk of overfitting, and (4) our approach doesn't assume visibility of test set images. That said, with sufficient resources, training the diffusion model more extensively and incorporating test images would likely improve results.

Best regards,
Haoyang
